IdentityCore/src/parameters/MSIDRequestParameters.m

```diff
@@ -245,4 +251,38 @@ - (BOOL)validateParametersWithError:(NSError **)error
     return YES;
 }
 
+#pragma mark - NSCopying
+
+- (instancetype)copyWithZone:(NSZone*)zone
```

Comment:
nit: should we also implement nscopying in interactive params?

Reply:
done